### PR TITLE
Remove insertion of ZWJ (200d)

### PR DESCRIPTION
**Pull request by TheEssemCraft**
In reference to https://bugzilla.mozilla.org/show_bug.cgi?id=1358240#c46

### CODE REVIEW

File: layerize.js [hunk 1]
```diff
@@ -577,14 +577,13 @@ function processFile(fileName, data) {
             chars.push({unicode: unicodes[0], components: layers});
         } else {
             // Ligatures: Keycap sequences only apply when there's U+FE0F in between.
-            // Otherwise, if not a Regional-Indicator pair or Fitzpatrick-modified char,
-            // insert ZWJ between components
+            // Twemoji already contains ZWJ characters
             if (unicodes.length == 2 && parseInt(unicodes[1], 16) == 0x20e3) {
                 unicodes = unicodes.join(",fe0f,").split(",");
```
Comment (timdream) [inline layerize.js]: Can you check if the keycap sequences come with 0xfe0f in their file names already? If so we could remove this `if` block.

File: layerize.js [hunk 2]
```diff
@@ -577,14 +577,13 @@ function processFile(fileName, data) {
...
             } else if (unicodes.length > 2 ||
                 !((parseInt(unicodes[0], 16) >= 0x1f1e6 && parseInt(unicodes[0], 16) <= 0x1f1ff) ||
                   (parseInt(unicodes[1], 16) >= 0x1f3fb && parseInt(unicodes[1], 16) <= 0x1f3ff))) {
-                unicodes = unicodes.join(",200d,").split(",");
+                // unicodes = unicodes.join(",200d,").split(",");
```
Comment (timdream) [inline layerize.js]: Let's remove this `if` block altogether.

**Comment by timdream** [comment]
According to GitHub you should rebase first. If you couldn't do it on GitHub, you should probably start over from a new branch.

File: layerize.js (hunk 1 above)
Comment (TheEssemCraft) [reply layerize.js]: It seems that they do.

**Comment by TheEssemCraft** [comment]
Sorry about the noob question, but how would I go about making a new branch based on the upstream repo? I tried to rebase/squash, but it failed, so this is my only option.

**Comment by timdream** [comment]
> Sorry about the noob question, but how would I go about making a new branch based on the upstream repo? I tried to rebase/squash, but it failed, so this is my only option.

I tried to figure out if I could create a pull request from `mozilla:master` into your `master` or `no-ci`, but both failed. Given that what we are looking at is quite small, the easiest way out now would probably delete your fork, fork again, and submit a pull request from a new branch of the new fork.

**Comment by TheEssemCraft** [comment]
The new pull request is at #24. Thanks for reminding me to never commit to master in a patch fork. :)

**Pull request closed without merging**